### PR TITLE
Updates to ServiceControl Containerization

### PR DESCRIPTION
**Pull request by poornimanayar**

### CODE REVIEW

File: servicecontrol/containerization/index.md [hunk 1]
```diff
@@ -16,32 +16,34 @@ ServiceControl is split into multiple docker images for each instance type. Thes
...
 | Docker for Windows                      | Yes       |                                                                    |
-| Docker (for Linux)                      | No        | ServiceControl can not run in Linux.                           |
+| Docker (for Linux)                      | No        | ServiceControl cannot run in Linux.                           |
 | Azure Container Services (ACS)          | No        | Works, but does not support persistent volumes for durable storage |
 | Azure Kubernetes Services (AKS)         | No        | Supports minimum of Windows Server 2019-based images               |
 | Amazon Elastic Container Services (ECS) | No        | Untested, guidance mentions AMIs available for Windows 2016        |
 | Amazon Elastic Kubernetes Service (EKS) | No        | EKS only supports Windows 2019                                     |
 
+Note: There is work in progress to enable ServiceControl to run in Linux containers. The details of the work are available on the [GitHub issue](https://github.com/Particular/ServiceControl/issues/3651).
```
Comment (poornimanayar) [inline servicecontrol/containerization/index.md:34]: Is it worth adding this here?

File: servicecontrol/containerization/index.md [hunk 2]
```diff
@@ -16,32 +16,34 @@ ServiceControl is split into multiple docker images for each instance type. Thes
...
+The `init` containers are similar to [installing all ServiceControl service types (regular, error, and monitoring) via the ServiceControl Management Utility or the Powershell Cmdlets](/servicecontrol/installation.md). This includes creating the database and the queues that ServiceControl uses. The `runtime` containers are similar to starting the corresponding Windows Services.
 
-Init containers are identical to the corresponding runtime image but with an added entrypoint arguments (`--setup`). It is also possible to use the runtime image and override the entrypoint and add this argument to achieve the same goal.
+`Init` containers are identical to the corresponding `runtime` image but with added entry point arguments (`--setup`). It is also possible to use the `runtime` image, override the entry point and add this argument to achieve the same goal.
```
Comment (poornimanayar) [inline servicecontrol/containerization/index.md:42]: Can we provide an example to enhance this?

File: servicecontrol/containerization/index.md (hunk 1 above)
Comment (mauroservienti) [reply servicecontrol/containerization/index.md:34]: Yes, it is 👍

File: servicecontrol/containerization/index.md (hunk 2 above)
Comment (mauroservienti) [reply servicecontrol/containerization/index.md:42]: It sounds like an implementation detail. I'm leaning toward removing it.

Comment (poornimanayar) [reply servicecontrol/containerization/index.md:42]: There is something similar in line 218 on this documentation which is what got me thinking

**Pull request merged**